pages/api/applications/[applicationId]/tasks/index.ts

```diff
@@ -104,16 +113,28 @@ async function handlePost(userId: string, req: NextApiRequest, res: NextApiRespo
 }
 
 function validateRequest(req: NextApiRequest): Nullable<MessageType> {
-  if (req.body.title && isEmpty(req.body.title)) {
+  if (req.body.title === undefined) {
+    return MessageType.MISSING_TITLE;
+  }
+
+  if (req.body.title === null) {
+    return MessageType.INVALID_TITLE;
+  }
+
+  if (isEmpty(req.body.title)) {
     return MessageType.EMPTY_TITLE;
   }
 
-  if (!isValidDate(req.body.dueDate)) {
+  if (req.body.dueDate === undefined) {
+    return MessageType.MISSING_DUE_DATE;
+  }
+
+  if (req.body.dueDate === null || !isValidDate(req.body.dueDate)) {
     return MessageType.INVALID_DUE_DATE;
   }
 
   if (
-    (req.body.notification || req.body.notificationDateTime !== undefined) &&
+    (req.body.notification === null || req.body.notificationDateTime !== undefined) &&
     !isValidDate(req.body.notificationDateTime)
```

Comment:
Are we doing the `isValidDate` check only when the date is not null and not undefined? In this case should it be like that? 

 ```suggestion
     (req.body.notification !== null && req.body.notificationDateTime !== undefined) &&
     !isValidDate(req.body.notificationDateTime)
```


I don't mind doing this either, it checks for both `null` and `undefined`.

 ```suggestion
     req.body.notification != null &&
     !isValidDate(req.body.notificationDateTime))
```